components/molecules/ContributorProfileHeader/contributor-profile-header.tsx

```diff
@@ -146,17 +146,29 @@ const ContributorProfileHeader = ({
             {/* Mobile dropdown menu */}
 
             <DropdownMenu>
-              <div className="flex items-center gap-2 mb-10 md:gap-6 flex-wrap">
-                <Button variant="primary" href={cardPageUrl(username!)}>
+              <div className="flex items-center md: gap-2 mb-10 md:gap-6 flex-wrap">
+                <Button className="sm:hidden" variant="primary" href={cardPageUrl(username!)}>
+                  <FaIdCard className="" />
+                </Button>
+                <Button className="hidden sm:inline-flex" variant="primary" href={cardPageUrl(username!)}>
                   <FaIdCard className="mt-1 mr-1" /> Get Card
                 </Button>
+
+                <Button
+                  onClick={() => handleCopyToClipboard(`${host}/user/${user?.user_metadata.user_name}`)}
+                  className="bg-white sm:hidden"
+                  variant="text"
+                >
+                  <FiCopy className="" />
+                </Button>
                 <Button
                   onClick={() => handleCopyToClipboard(`${host}/user/${user?.user_metadata.user_name}`)}
-                  className="px-8 py-2 bg-white "
+                  className="px-8 py-2 bg-white hidden sm:inline-flex"
                   variant="text"
                 >
                   <FiCopy className="mt-1 mr-1" /> Share
                 </Button>
```

Comment:
You can replicate the responsiveness for this `Button` too, instead of a second button.

Reply:
Hi @babblebey ,
I had tried your solution, but unfortunately with the `span`, I would have had to reapply the CSS classes on that as well. The problem with this approach is that if there were changes to the design in the future, such as the text of the buttons becoming bold, it would break here, and someone would have to modify this component as well.

To avoid button duplication, one could consider using a JavaScript method to determine the current breakpoint. However, this would deviate from the focus of issue.